src/baldr/graphreader.cc

```diff
@@ -880,27 +892,31 @@ int GraphReader::GetTimezone(const baldr::GraphId& node, const GraphTile*& tile)
   return (tile == nullptr) ? 0 : tile->node(node)->timezone();
 }
 
+std::shared_ptr<valhalla::IncidentsTile> GraphReader::GetIncidentTile(const GraphId& tile_id) const {
+  return enable_incidents_ ? incident_singleton_t::get(tile_id.Tile_Base())
```

Comment:
this had to move from the header to the implementation file because of accessing the singleton